sample/rules_test.go

```diff
@@ -1392,12 +1403,13 @@ func TestRulesDatatypes(t *testing.T) {
 				{
 					Event: types.Event{
 						Data: map[string]interface{}{
-							"test": 9.3,
+							"test": 9.3, // "9.3" is greater than "10.3" when compared as a string
```

Comment:
👀

Reply:
Huh. Turns out it's consistent with F# too, so this isn't just some "rob pike is a weird dude" behavior:

```
λ ~/scratch/otel-pixie/ dotnet fsi

Microsoft (R) F# Interactive version 12.4.0.0 for F# 7.0
Copyright (c) Microsoft Corporation. All Rights Reserved.

For help type #help;;

> "9.3" > "10.3";;
val it: bool = true
```

Reply:
Yeah, alphabetically '9' is greater than '1'. :)